src/dev/typescript/run_type_check_cli.ts

```diff
@@ -80,7 +80,8 @@ export async function runTypeCheckCli() {
     process.exit();
   }
 
-  await buildRefs(log);
+  await buildRefs(log, 'tsconfig.refs.json');
+  await buildRefs(log, 'x-pack/tsconfig.refs.json');
```

Comment:
I hardcoded the path here since this script is meant to be used in dev only. @elastic/kibana-operations any problems with this?